docs/reference/mapping/types/feature-vector.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[feature-vector]]
+=== Feature vector datatype
+
+A `feature_vector` field can index numeric feature vectors, so that they can
+later be used to boost documents in queries with a
+<<query-dsl-feature-query,`feature`>> query.
+
+It is analogous to the <<feature,`feature`>> datatype but is better suited
+when the list of features is sparse so that it wouldn't be reasonable to add
+one field to the mappings for each of them.
+
+[source,js]
+--------------------------------------------------
+PUT my_index
+{
+  "mappings": {
+    "_doc": {
+      "properties": {
+        "topics": {
+          "type": "feature_vector" <1>
+        }
+      }
+    }
+  }
+}
+
+PUT my_index/_doc/1
+{
+  "topics": { <2>
+    "politics": 20,
+    "economics": 50
+  }
+}
+
+PUT my_index/_doc/2
+{
+  "topics": {
+    "politics": 5,
+    "sports": 80
+  }
+}
+
+GET my_index/_search
+{
+  "query": {
+    "feature": {
+      "field": "topics.politics"
+    }
+  }
+}
+--------------------------------------------------
```

Comment:
I wonder if at least one of the example values above should be a fractional value to make it clearer that the field type accepts float values and not just integer values? It might also be worth adding a note to this effect below?

Reply:
Agreed, @gibrown had a similar comment